pages/docs/clients/jdbc.html

```diff
@@ -4,13 +4,56 @@
 
 ## Compatibility
 
-Each major version of the Crate JDBC driver is bound to a major version
-of Crate. Find out which drivers are compatible to your Crate version with this [compatibility list](https://github.com/crate/crate-jdbc#compatibility).
+### Crate JDBC driver version 1.x.x:
+
+Crate JDBC driver 1.x.x requires a Crate version greater or equal than
+*0.38.0* but lower than *0.56.0*. Each major version of the driver is bound
+to the major version of Crate. Below, you can see the compatibility table
+for the JDBC driver version 1.x.x:
+
+| Crate JDBC driver | Crate    |
+|-------------------|----------|
+| *1.8.X*           | *0.49.X* |
+| *1.9.X*           | *0.51.X* |
+| *1.10.X*          | *0.54.X* |
+| *1.11.X*          | *0.54.X* |
+| *1.12.X*          | *0.54.X* |
+| *1.13.X*          | *0.55.X* |
+
+### Crate JDBC driver version 2.x.x
+
+Version 2.x.x of the Crate JDBC driver uses the
+[PostgreSQL Wire Protocol](https://www.postgresql.org/docs/current/static/protocol.html).
```

Comment:
I think we should add a note somewhere here which says that the they need to make sure to use the postgres port.